tools/downloader/topologies_test.py

```diff
@@ -0,0 +1,79 @@
+import unittest
+import os
+import numpy as np
+import cv2 as cv
+
+import topologies as omz
+
+from pathlib import Path
+from openvino.inference_engine import IECore
+
+class TestTopologies(unittest.TestCase):
+
+    download_path = Path(os.getenv('OPEN_MODEL_ZOO_MODELS_DIR', Path('.').resolve() / 'models'))
+    ssd = omz.load_model('mobilenet-ssd')
+
+    def test_load_model(self):
+        self.assertTrue(os.path.exists(
+            os.path.join(self.download_path, 'public', 'mobilenet-ssd', 'mobilenet-ssd.caffemodel')))
```

Comment:
I'd like to recommend to remove this test - it's not so critical which structure has downloading folder but other tests will fail if there is something wrong with it. (I just want to remove `getenv` from the test 🐤 )